qiskit/providers/ibmq/ibmqbackendservice.py

```diff
@@ -35,21 +36,42 @@
 
 
 class IBMQBackendService(SimpleNamespace):
-    """Backend namespace for an IBM Quantum Experience account provider."""
+    """Backend namespace for an IBM Quantum Experience account provider.
+
+    Represent a namespace for the IBM Quantum Experience backends available
+    to this account. This namespace is used to override the
+    :meth:`backends<AccountProvider.backends>` method for convenience.
+
+    It is still possible to access all the backends this account has access to in an exact
+    manner::
+
+        backends = provider.backends()
+
+    However, you are now able to retrieve jobs based on your account, instead of retrieving them
```

Comment:
Be careful with the wording here. `account` can easily be interpreted as an IQX account, which may include multiple providers. Instead it's probably easier to just say "you can now retrieve jobs from a provider without specifying the backend name".